plugin.js

```diff
@@ -2,13 +2,21 @@
 
 const fp = require('fastify-plugin')
 const cookie = require('cookie')
+const cookieSignature = require('cookie-signature')
+
+let signSecret = ''
```

Comment:
I think it would be better to attach this to the request, e.g.:

```js
const sym = Symbol.for('fastify-cookie.secret')

function fastifyCookieSetCookie (name, value, options) {
  if (opts.signed) cookieSignature.sign(value, this[sym])
}

function plugin (fastify, options, next) {
  fastify.decorateRequest(sym, options.secret)
}
```

Reply:
Great, thanks.